examples/md-flexible/src/ParticleSerializationTools.cpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file ParticleSerializationTools.cpp
+ * @author J. Körner
+ * @date 13.05.2021
+ */
+#include "ParticleSerializationTools.h"
+
+namespace ParticleSerializationTools {
+void serializeParticle(const ParticleType &particle, std::vector<char> &serializedParticles) {
+  ParticleAttributes attributes{.position{particle.getR()},
+                                .velocity{particle.getV()},
+                                .force{particle.getF()},
+                                .id = particle.getID(),
+                                .ownershipState = particle.getOwnershipState(),
+                                .typeId = particle.getTypeId(),
+                                .oldForce{particle.getOldF()}};
```

Comment:
these use dedicated initialization which is a c++20 extension (and previously a gnu extension).
while I see the benefit, they sadly are not yet usable, because we still use c++17.

Reply:
the struct ParticleAttributes has been removed in branch md-flex/particle-serialization which will be reviewed after md-flex/cart-decomposition has been merged, as it depends on it.

Reply:
PRs will not be merged with failing compilation...

Reply:
as it stands, #630's target is this branch anyways. so it might make sense to merge that one first into this branch.